src/app/metadata/features/dataset-details/dataset-details.html

```diff
@@ -224,10 +224,50 @@ <h2 class="inline-flex items-center text-2xl">
                   ><mat-icon inline fontIcon="data_loss_prevention"></mat-icon></span
                 ><span>Policy and Data Access Committee</span>
               </h2>
-              <p class="text-base">
-                <strong>Data Access Committee:</strong> {{ dac().alias }}
-              </p>
-              <p class="text-base"><strong>e-Mail:</strong> {{ dac().email }}</p>
+              <div
+                class="*:text-base lg:grid lg:grid-cols-2 lg:gap-x-2 2xl:grid-cols-3"
+              >
+                <p><strong>Data Access Committee:</strong> {{ dac().alias }}</p>
+                <p><strong>e-Mail:</strong> {{ dac().email }}</p>
+              </div>
+              <div
+                class="*:-mt-1.5 *:text-base lg:grid lg:grid-cols-2 lg:gap-x-2 2xl:grid-cols-3"
+              >
+                <p>
+                  <strong>Data Use Permission:</strong>&nbsp;<a
+                    appExtLink
+                    [href]="
+                      `http://purl.obolibrary.org/obo/${(dap().data_use_permission_id | replaceStringPipe: ':' : '_')}`
+                    "
+                    class="lowercase"
+                    >{{ dap().data_use_permission_term | underscoreToSpace }}</a
+                  >
+                </p>
+                <p>
+                  <strong>Data Use Modifiers:</strong>
+                  @if (dap().data_use_modifier_terms.length > 0) {
+                    @for (
+                      term of dap().data_use_modifier_terms;
+                      track term;
+                      let idx = $index;
+                      let first = $first;
+                      let last = $last
+                    ) {
+                      {{ first ? ' ' : '' }}
+                      <a
+                        appExtLink
+                        [href]="
+                          `http://purl.obolibrary.org/obo/${(dap().data_use_modifier_ids[idx] | replaceStringPipe: ':' : '_')}`
```

Comment:
Would be better to avoid code duplication and hardcoding the url in the template. The `dap()` signal could return the actual data policy object, enriched with the URLs computed from the IDs. That way you also wouldn't need the replaceStringPipe.

The base URL should be defined once as a constant. I think we should use `https` instead of `http`. The latter is the canonical form, but the former works as well, and avoids "mixed content" or other warnings when linking from https to http in the browser.